patches/chrome-browser-ui-startup-startup_browser_creator.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/browser/ui/startup/startup_browser_creator.cc b/chrome/browser/ui/startup/startup_browser_creator.cc
+index f6f46c4dc501f9f8b0cc72267e78a611452d97cd..4a011ffb076aed721134eb4c4e17b0564d9d9375 100644
+--- a/chrome/browser/ui/startup/startup_browser_creator.cc
++++ b/chrome/browser/ui/startup/startup_browser_creator.cc
+@@ -289,6 +289,7 @@ void DumpBrowserHistograms(const base::FilePath& output_file) {
+ // Returns whether |profile_info.profile| can be opened during Chrome startup
+ // without explicit user action.
+ bool CanOpenProfileOnStartup(StartupProfileInfo profile_info) {
++BRAVE_CAN_OPEN_PROFILE_ON_STARTUP
```

Comment:
can't we just set this profile as `StartupProfileMode::kProfilePicker`?

Reply:
I could look into overriding `GetStartupProfilePath` but looks a little more complicated. Will have to spend some more time tracing.